### DISCUSSION TITLE
[FEATURE] 深色自动切换

### DISCUSSION THREAD

**Discussion by inuyasha-660** [hugo-fixit/FixIt#371] [Q&A]
### Describe the feature you want 描述你的功能需求

希望能添加自动切换深色，比如18:00~8:00为深色，其余时间为明亮

**Comment by Lruihao** [comment]
当设置 defaultTheme = "auto" 时，主题颜色随着系统的改变而改变（`prefers-color-scheme`）。

系统设置例如：

- Mac：设置 => 外观 => 自动
- iPhone: 设置 => 显示与亮度 => 自动
- ...
